src/installer/installer.go

```diff
@@ -62,6 +63,46 @@ type installer struct {
 	kcBuilder       k8s_client.K8SClientBuilder
 }
 
+type FirstIgnition struct {
```

Comment:
bm-inventory now needs to pass the hostname so we can update the first ignition? 
Why do it here and not in bm-inventory where this ignition is generated in the first place?

Reply:
ignitions are not hosts specific, and hostnames are, so that's why we do on the node